app/views/assignments/show.html.erb

```diff
@@ -1,3 +1,4 @@
+<div>
 <div class="container">
```

Comment:
Is the opening tag required ? I think it is just due to one container div not getting closed, try with just the closing tag.